src/reducers/index.js

```diff
@@ -12,9 +12,9 @@ const initialState = {
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
-    // Fill in the body of this case
+      return {...state, count: initialState.count++}
```

Comment:
Go to next line.

return {
    ...state,
    count: initialState.count++
}

Reply:
Ok, fixed